sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/KnownLinuxVirtualMachineImage.java

```diff
@@ -8,6 +8,12 @@ public enum KnownLinuxVirtualMachineImage {
     UBUNTU_SERVER_16_04_LTS("Canonical", "UbuntuServer", "16.04.0-LTS"),
     /** UbuntuServer 18.04LTS. */
     UBUNTU_SERVER_18_04_LTS("Canonical", "UbuntuServer", "18.04-LTS"),
+    /** UbuntuServer 18.04LTS Gen2. */
+    UBUNTU_SERVER_18_04_LTS_GEN2("Canonical", "UbuntuServer", "18_04-lts-gen2"),
+    /** UbuntuServer 20.04LTS. */
+    UBUNTU_SERVER_20_04_LTS("Canonical", "0001-com-ubuntu-server-focal", "20_04-lts"),
+    /** UbuntuServer 20.04LTS Gen2. */
+    UBUNTU_SERVER_20_04_LTS_GEN2("Canonical", "0001-com-ubuntu-server-focal", "20_04-lts-gen2"),
```

Comment:
Seems for ubuntu20.04 and later, they changed offer name from `UbuntuServer` to `0001-com-ubuntu-server-focal`:

https://github.com/Azure/azure-cli/issues/13320#issuecomment-649867249

https://discourse.ubuntu.com/t/find-ubuntu-images-on-microsoft-azure/18918

Reply:
Trusted launch is only supported on gen2 VMs and images, hence the UBUNTU_SERVER_18_04_LTS_GEN2 and UBUNTU_SERVER_20_04_LTS_GEN2.

UBUNTU_SERVER_20_04_LTS is for gen1 pairing with added gen2.